cmd/geth/testdata/bls-account-usage-demo.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+echo "0. prepare---------------------------------------------------------------------------------\n\n"
+echo 123abc >bls-password.txt
+echo 123abc7890 > bls-account-password.txt
+basedir=$(cd `dirname $0`; pwd)
+workspace=${basedir}/../../../
+
+echo "1. create a bls account--------------------------------------------------------------------\n\n"
+${workspace}/build/bin/geth bls account new --blsaccountpassword ./bls-account-password.txt --blspassword ./bls-password.txt --datadir ./bls
```

Comment:
It can be confusing to use two types of passwords. It's better to use just one.

Reply:
accept